onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc

```diff
@@ -526,7 +526,7 @@ TEST(ConvTransposeTest, ConvTranspose_InvalidKernelShape) {
                       // so drop the part that differs from the expected string
                       "kernel_shape num_dims is not compatible with W num_dims. kernel_shape: {1,1,1,5} W: {1,1,",
                       {kTensorrtExecutionProvider, kQnnExecutionProvider,
-                       kDmlExecutionProvider});  // TODO: Unskip when fixed #41968513
```

Comment:
Would you like to comment on the reason for exclusion?

Reply:
This is caused by the latest OV commits add some logic to make this pattern work. But V2025.4 can pass it. I will report a new issue to get this test pass in the next release.